INSTRUMENTATION_GUIDE.md

```diff
@@ -47,15 +47,15 @@ When `LOG_DIR` is set, each run creates a detailed log file with all INFO-level
 ### Basic Usage (No Logging)
 
 ```bash
-echo "What is 46048 × 42098?" | python3 multiagent_reasoner.py
+echo "What is 46048 × 42098?" | python3 decomposer/multiagent_reasoner.py
```

Comment:
Update INSTRUMENTATION_GUIDE.md with the fact that the multiagent_reasoner.py stuff has moved inside decomposer package.